lib/bootstrap/bundler.rb

```diff
@@ -95,6 +95,19 @@ def invoke!(options = {})
 
       require "bundler"
       require "bundler/cli"
+
+      # create Gemfile from template iff it does not exist
+      unless ::File.exists?(Environment::GEMFILE_PATH)
+        ::FileUtils.copy(
+          ::File.join(ENV["LOGSTASH_HOME"], "Gemfile.template"), Environment::GEMFILE_PATH
+        )
+      end
+      # create Gemfile.jruby-1.9.lock from template iff a template exists it itself does not exist
+      lock_template = ::File.join(ENV["LOGSTASH_HOME"], "Gemfile.jruby-1.9.lock.release")
+      if File.exists?(lock_template) && !::File.exists?(Environment::LOCKFILE)
+        ::FileUtils.copy(lock_template, Environment::LOCKFILE)
```

Comment:
Can we remove the double `::` before the FileUtils? it's actually needed for the `::File` module because we have a plugin that clash the namespace (LogStash::Inputs::File)